martin/src/pg/query_tables.rs

```diff
@@ -217,10 +233,18 @@ async fn calc_bounds(
     table: &str,
     geometry_column: &str,
     srid: i32,
+    is_quick: bool,
 ) -> PgResult<Option<Bounds>> {
-    Ok(pool.get()
-        .await?
-        .query_one(&format!(
+    let sql = if is_quick {
+        let schema = escape_literal(schema);
+        let table = escape_literal(table);
+        let geometry_column = escape_literal(geometry_column);
```

Comment:
@nyurik  any help on this? I think we should call `escape_literal` before `ST_EstimatedExtent`, but the tests still fails.
`[2024-03-13T04:33:08Z ERROR martin::pg::builder] Failed to create a source: Postgres error while querying table bounds: db error: ERROR: invalid name syntax`

Reply:
And the SQL is : `SELECT ST_EstimatedExtent('"Quotes'' and Space.Dot.', '. Points" ''quote', '. "Geom"') as bounds`

Reply:
Sigh, one of those painful cases I guess - I created a bug https://trac.osgeo.org/postgis/ticket/5697 

So to move forward, I think we will have to do a workaround with the following logic:
* if schema, table, and geom column is "simple" (matches this regex for all 3 values:  `^[a-zA-Z_][a-zA-Z_0-9]*$`), pass these values "as is", but make sure to pass them as parameters (second param in `.query_one()`)
* if not, use the existing query